ts/utils/configs.ts

```diff
@@ -8,6 +8,9 @@ export const configs = {
     BACKEND_BASE_PROD_URL: 'https://website-api.0x.org',
     BACKEND_BASE_STAGING_URL: 'https://staging-website-api.0x.org',
     BACKEND_BASE_DEV_URL: 'https://localhost:3001',
+    API_BASE_PROD_URL: 'https://api.0x.org',
+    API_BASE_STAGING_URL: 'https://staging.api.0x.org',
+    API_BASE_DEV_URL: 'http://localhost:3000',
```

Comment:
NIT: I think `website-backend` defaults to using port `3000` for http, would be nice if they don't conflict when running locally.